src/NuGet.Indexing/VersionAnalyzer.cs

```diff
@@ -1,15 +1,30 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-using Lucene.Net.Analysis;
+
 using System.IO;
+using Lucene.Net.Analysis;
 
 namespace NuGet.Indexing
 {
     public class VersionAnalyzer : Analyzer
     {
+        private readonly bool _caseSensitive;
+
+        public VersionAnalyzer(bool caseSensitive)
+        {
+            _caseSensitive = caseSensitive;
+        }
+
         public override TokenStream TokenStream(string fieldName, TextReader reader)
         {
-            return new SemanticVersionFilter(new KeywordTokenizer(reader));
+            TokenStream stream =  new SemanticVersionFilter(new KeywordTokenizer(reader));
```

Comment:
nit: `var`?

Reply:
`LowerCaseFilter` cannot be assigned to `SemanticVersionFilter`.